flink-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -176,5 +176,23 @@ under the License.
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-table_2.10</artifactId>
+			<version>${project.version}</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-scala_2.10</artifactId>
```

Comment:
This is interesting as now `flink-table_2.10` no longer pulls in `flink-streaming-scala_2.10` as a transitive dependency. Therefore the change needs to introduce the dependency in order to compile anything against the `flink-streaming-scala` module (e.g., `StreamExecutionEnviornment`).